docs(express): add note about npm vulnerability warning (#1174)
canonical/rockcraft: docs/tutorial/express.rst

    :end-before: [docs:init-app-end]
    :dedent: 2

.. note::

>    After ``npm install``, npm may report known vulnerabilities in installed packages because it performs a security scan of downloaded dependencies.

Reviewer: Nitpick:
```suggestion
   During the ``npm install`` step, npm performs a security scan of downloaded dependencies, and it warns about any known vulnerabilities in installed packages.
```
My suggestion tries to address the sequence of events in this sentence (AFAIK the security scan happens as the final part of the ``npm install`` step, but the sentence implies it happens after -- let me know if I'm mistaken!). My attempt was to describe the events in the order in which they occur. 

I also used the verb "warns" to connect this sentence with the next one.
